### DISCUSSION TITLE
Pyspark support

### DISCUSSION THREAD

**Discussion by raghuvarranvh** [deepjavalibrary/djl#1017] [Development]
Does DJL support development in pyspark? Any tutorials for this?

**Comment by lanking520** [comment]
You can try to follow the general instruction with Py4J https://stackoverflow.com/questions/33544105/running-custom-java-class-in-pyspark. 

DJL itself is a Java package, it can work perfectly well with Spark Scala. If you plan to use DJL in PySpark, you could try to build a Scala SPARK UDF with DJL and then expose that in your PySpark application to call.
